packages/components/src/util/clickable.tsx

```diff
@@ -20,7 +18,7 @@ export type ClickableProps<IComponent extends React.ElementType> = {
   /**
    * A hidden prop for visual testing
    */
-  state?: "inactive" | "hover" | "focus" | "active" | "disabled";
```

Comment:
Could you help me understand the changes around "disabled" in this PR?

Reply:
I think the styles are now based off of the `:disabled` pseudo-class, instead of an explicit class.